engine/src/witness/sol/nonce_witnessing.rs

```diff
@@ -57,7 +57,7 @@ impl<Inner: ChunkedByVault> ChunkedByVaultBuilder<Inner> {
 			let state_chain_client = state_chain_client.clone();
 			async move {
 				let nonce_accounts: HashMap<SolAddress, SolHash> = state_chain_client
-					.storage_map::<pallet_cf_environment::SolanaUnAvailableNonceAccounts<
```

Comment:
I guess this was a typo

Reply:
Just double checked - the typo is now fixed